src/screens/Home.js

```diff
@@ -29,18 +31,36 @@ export default ({
   navigation,
   //state props
   categories,
+  learntPhrases,
   nativeLanguage,
   //actions
+  setPhrases,
   setCategories,
   setCurrentCategory,
-  setPhrases,
+  setLearntCategories,
+  setCurrentLearntCategory,
 }) => {
+  // const saveData = async list => {
+  //   try {
```

Comment:
We don't want to have all these comments in the production code, besides, it seems like this is supposed to be a PR about switching languages, not the learnt phrases. It would be better to turn it into two separate PRs each with only the relevant changes